lms/envs/common.py

```diff
@@ -950,6 +950,8 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 SESSION_COOKIE_SECURE = False
 SESSION_SAVE_EVERY_REQUEST = False
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.PickleSerializer'
+SESSION_COOKIE_SAMESITE = 'None'
+SESSION_COOKIE_SAMESITE_FORCE_ALL = True
```

Comment:
If these same settings will live with future Django 2.2.x versions then better to use prefix `DCS` to avoid conflicts as per https://github.com/jotes/django-cookies-samesite/issues/19

Reply:
Im not sure if adding `DCS` will introduce errors with 1.11.x
Maybe we can consider this while migrating to 2.2.x?

Reply:
It won't as settings are checked for both https://github.com/jotes/django-cookies-samesite/blob/master/django_cookies_samesite/middleware.py#L31 but lets keep it for later